src/api.jl

```diff
@@ -73,7 +73,7 @@ Initializes the Optimizer library. This must be called before any other library
 
 """
 function init()
-    @checked Lib.XPRSinit(C_NULL)
+    Lib.XPRSinit(C_NULL)
```

Comment:
We could documente in the macro def why some functions wont use it.

Reply:
Resolved in https://github.com/jump-dev/Xpress.jl/pull/126/commits/f7fc8fb66bf26225aa680d6e65a2a06dbad89a69.